packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [20.0.0]
+
+### Removed
```

Comment:
Missing change entries:

```suggestion
### Changed

- Bump `@metamask/base-controller` from `^7.0.1` to `^7.0.2` ([#4862](https://github.com/MetaMask/core/pull/4862))
- Bump `@metamask/json-rpc-engine` from `^10.0.0` to `^10.0.1` ([#4862](https://github.com/MetaMask/core/pull/4862))

### Removed
```

Reply:
Could we update the release script to automatically generate these since they follow a standard format and can be derived from the `package.json` diffs?

Reply:
Certainly. They'd also be easier to spot if we started adding them to the `Unreleased` section upfront, rather than updating changelogs later on as part of the RC.